curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -141,19 +140,18 @@ void FuseS3Client::fetchDataEnqueue(fuse_ino_t ino) {
                     << ", inodeid = " << ino;
             return;
         }
-        google::protobuf::Map<uint64_t, S3ChunkInfoList> *s3ChunkInfoMap
-          = nullptr;
-        {
-            ::curve::common::UniqueLock lgGuard = inodeWrapper->GetUniqueLock();
-            s3ChunkInfoMap = inodeWrapper->GetChunkInfoMap();
-        }
-        if (nullptr == s3ChunkInfoMap ||
-          s3ChunkInfoMap->empty()) {
+        uint64_t fileSize = inodeWrapper->GetLength();
+        std::unique_ptr<char[]> buffer(new char[fileSize]);
```

Comment:
if fileSize is too large, the new may fail

Reply:
there are maybe many files , and file maybe large too, so the new() for read()  may cause memory problem